contents/handbook/people/team-structure/platform/objectives.mdx

```diff
@@ -1,11 +1,42 @@
-* **Proposed Objective**:
-    * PostHog Cloud in the US remains reliable and launch EU Cloud
-* **Proposed Key Results**:
-    * Multiple organizations using EU hosted PostHog Cloud without regressions in either EU or US cloud
-    * Reliable monitoring and alerting on all key services
-    * Upgrade to Kubernetes 1.25 to unblock key customers
-    * Meet SOC 2 compliance requirements for all cloud infrastructure
-* **Rationale**:
-    * To nail self-serve we need to make it as easy as possible for customers to sign up and get started
-    * For businesses in the EU today, the main barrier is needing to self host to keep data in the EU, this limits trust in our brand and the potential adoption we can drive
-    * Shipping a secure cloud in the EU will also encourage many businesses to get started with PostHog Cloud, even if they ultimately intend to self-host
\ No newline at end of file
+- Objective: High uptime and increase the resilience/redundancy
+    - Key results:
+        - Team to fill in
```

Comment:
1) Migrate Postgres out of Heroku
  - This will not scale into the future and doing it before it becomes a huge blocking problem is best. Likely to be very awkward + difficult without downtime
2) Regular synthetic load testing on prod
  - Give us a ceiling for what we _know_ we can handle
  - Give teams real numbers for performance and stability improvements
  - 100% automated, can watch prometheus for an increase in error rate etc

Reply:
> Likely to be very awkward + difficult without downtime

I'm fine with app downtime during a migration, if we can figure out how to continue ingesting events into a DLQ or whatever for that period.